docs/UI/Slider/index.md

```diff
@@ -29,6 +29,7 @@ Purpose
 |Name|Type|Mandatory|Additional|
 |:---|:---|:--------|:---------|
 |sliderPosition|Integer|false|minvalue: 1<br>maxvalue: 26|
+|cancelID|Integer|false||
```

Comment:
Should be in the request, not the response

Reply:
fixed in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/163/commits/934dd94cbb0be99702d7e8dc50d6401f083a173b